yorkie/src/main/kotlin/dev/yorkie/document/crdt/RHTPQMap.kt

```diff
@@ -0,0 +1,161 @@
+package dev.yorkie.document.crdt
+
+import android.util.Log
+import dev.yorkie.document.time.TimeTicket
+import dev.yorkie.util.MaxPriorityQueue
+import dev.yorkie.util.PQNode
+
+
+/**
+ * RHTPQMap is replicated hash table with priority queue by creation time.
+ *
+ * @internal
+ */
+internal class RHTPQMap {
+    private val logTag = "RHTPQMap"
+
+    private val elementQueueMapByKey: MutableMap<String, MaxPriorityQueue<PQNode<TimeTicket, CrdtElement>>> =
+        mutableMapOf()
+    private val nodeMapByCreatedAt: MutableMap<String, RHTPQMapNode> = mutableMapOf()
```

Comment:
Can we use `TimeTicket` directly instead of `string` here? 🤔

```suggestion
    private val nodeMapByCreatedAt: MutableMap<TimeTicket, RHTPQMapNode> = mutableMapOf()
```

Reply:
Done : [6559d1f](https://github.com/yorkie-team/yorkie-android-sdk/pull/10/commits/6559d1f87644f6f857cae127e373f017ed85c6d8)